makani/models/preprocessor.py

```diff
@@ -427,6 +427,13 @@ def get_internal_rng(self, gpu=True):
                 return self.input_noise.rng_cpu
         else:
             return None
+    
+    def set_rng(self, reset = True, seed=333):
```

Comment:
what is the logic behind the reset argument here? Shouldn't the userjust avoid calling set_rng if he does not want to overwrite it?

Reply:
A call to update should always change the noise state, and for stateful noise we want to distinguish between generating a new noise source or doing an update.

Reply:
I am indifferent about it. If the user knows what they are doing, I am happy with that.